admin/views/test-developer/assign-forms.ejs

```diff
@@ -0,0 +1,109 @@
+                </p>
+            </div>
+
+            <% if (Array.isArray(checkFormsList)) { %>
+            <table class="spacious" id="assignFormToWindowList" role="group">
+                <thead>
+                    <tr>
+                        <th scope="col">Form name</th>
+                        <th scope="col">
+                            <div class="tick-all-checkboxes-wrapper">
+                                <label class="tick-all-checkboxes" id="selectAll" for="tickAllCheckboxes">Select all</label>
+                                <label class="tick-all-checkboxes all-hide" id="deselectAll" for="tickAllCheckboxes">Deselect all</label>
+                                <div class="multiple-choice-mtc">
+                                    <input id="tickAllCheckboxes" name="allForms" type="checkbox" aria-label="Select all Forms.">
+                                    <div></div>
+                                </div>
+                            </div>
+                        </th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <% checkFormsList.forEach(checkForm => { %>
+                    <tr>
+                        <td>
+                            <label for="checkForm<%= checkForm._id %>">
+                                <%= checkForm.name %>
+                            </label>
+                        </td>
+                        <td class="align-right">
+                            <div class="multiple-choice-mtc">
+                                <input id="checkForm<%= checkForm._id %>"
+                                       name="checkForm[<%= checkForm._id %>]"
+                                       type="checkbox"
+                                       value="<%= checkForm._id %>"
+                                       aria-label="Tick form <%= checkForm.name %>"
+                                       aria-checked="false"
+                                       role="checkbox">
+                                <div></div>
+                            </div>
+                        </td>
+                    </tr>
+                    <% }) %>
+                </tbody>
+            </table>
+            <input type="hidden" name="checkWindowName" value="<%= checkWindowName %>">
+            <input type="hidden" name="checkWindowId" value="<%= checkWindowId %>">
+            <% } %>
+
+            <div class="back-to-top-container">
+                <a href="#top">Back to top</a>
+            </div>
+
+        </div>
+
+        <div class="column-one-third add-title-margin">
+            <aside role="complementary" class="support-column">
+                <h3 class="heading-medium">Related</h3>
+                <nav role="navigation">
+                    <ul class="list">
+                        <li>
+                            <a href="/PDFs/MTC_administration_guidance_June-2017-trial.pdf" target="_blank">Guidance</a>
```

Comment:
Needs rel="noopener noreferrer"  for target="_blank"
https://paper.dropbox.com/doc/Links-xiWNcJk5Wpb8Sj6G4Prtl

Reply:
Added to all the links with target="_blank" of this feature.